### PR TITLE
fix: illogical naming and removed incomplete experiments

### LINKED ISSUE #876: Rename illogical headers
**Actual Behaviour**

`Saved experiments` doesn't make sense as they are not user saved experiments.

**Expected Behaviour**

Similar headings need to be changed

**Steps to reproduce it**

1. Look at the application headers

**LogCat for the issue**

> Not available

**Screenshots of the issue**

> Not available

**Would you like to work on the issue?**

-

### PR DESCRIPTION
**Pull request by CloudyPadmal**
Fixes #876 

Changes: 
- Changed `Applications` to `Instruments`
- Took out `Design Experiments` as it is just a blank screen
- Took out `Experiments` as they are just documentation
- Took out `Wave Generator`
- Now `Instruments` is in the starting screen

Screenshots for the change: 
device-2018-05-16-220305

APK for testing:
[app-debug.apk.zip](https://github.com/fossasia/pslab-android/files/2009989/app-debug.apk.zip)

### CODE REVIEW

**Comment by Avjeet** [comment]
I think you should remove the wavegen also for now as it is empty activity I will push the complete activity when its implementation and layout are done

**Comment by abhinavraj23** [comment]
@Avjeet I think for the wavegan part you can change that in your PR.

**Comment by CloudyPadmal** [comment]
@Avjeet wavegen implementation wouldn't take that long right?

**Comment by Avjeet** [comment]
But currently I don't have the device so I cant test the code which I write in wavegen, so till then remove the wavegen and I will push the complete code in once.

File: app/src/main/res/menu/activity_main_drawer.xml [hunk 1]
```diff
@@ -6,18 +6,9 @@
             android:icon="@drawable/ic_home_black_24dp"
             android:title="@string/nav_home" />
```
Comment (Avjeet) [inline app/src/main/res/menu/activity_main_drawer.xml]: make this "Device"

**Comment by Avjeet** [comment]
can u plz merge this PR so that we can work ahead of this without conflicts in code

**Comment by harsh-2711** [comment]
@CloudyPadmal Please merge this PR so I can work on changing label Devices from Home without any merge conflicts

**Comment by CloudyPadmal** [comment]
I cannot merge it without it's getting approved.

**Pull request merged**